mayavi/tools/tools.py

```diff
@@ -304,12 +304,12 @@ def set_extent(module, extents):
     extenty = 0.5 * (ymax - ymin)
     extentz = 0.5 * (zmax - zmin)
     # Now the actual bounds.
-    xmin, xmax, ymin, ymax, zmin, zmax = module.actor.actor.bounds
+    xmin, xmax, ymin, ymax, zmin, zmax = module.actor.bounds
```

Comment:
I think this will need to be conditional.  For example one could do the following:
```
actor = getattr(module.actor, 'actor', module.actor)
```
and then use `actor.bounds` etc.  The reason is that in some cases the module.actor may not be an image actor, so one should test that for the case when the object is not an Image actor that it continues to work.